modules/get-started/pages/quick-start.adoc

```diff
@@ -36,111 +57,301 @@ docker compose version
+. Enter "Sasha" in the username field.
+// (step {"action":"find", "selector": "[data-testid='create-user-name']", "typeKeys": "Sasha", "click": true})
+. Enter "sashapassword" in the password field.
+// (step {"action":"find", "selector": "[data-testid='password-input-toggle']", "click": true})
+// (step {"action":"find", "selector": "[data-testid='test_field']", "setVariables": [{"name": "password","regex": ".*"}], "click": true})
+// (step {"action":"find", "selector": "[data-testid='password-input-toggle']", "click": true})
+// (step {"action":"wait"})
+// (step {"action":"saveScreenshot", "path": "user.png", "directory": "../../modules/console/images", "overwrite": "byVariation"})
+. Click *Create*.
+
+image::console:user.png[]
+
+You now have a new user called Sasha. But this user has no permissions yet.
+
+. In the *ACLs* tab, click *Create ACL*.
+. Enter "Sasha" in the user/principal field.
+. In the topic selector dropdown, select *Literal* and enter "transactions" in the input field.
+. Under *Operations*, click the *All* dropdown and select *Allow*.
+. Scroll down and click *OK*.
+
+Now Sasha has full access only to the topic called transactions.
+
+To test these permissions, you can use `rpk` to connect to Redpanda as the user Sasha.
+
+. Try to access the `logins` topic as Sasha:
 +
-.Reveal the YAML content
-[%collapsible]
-====
-.`docker-compose.yml`
-[,yaml,subs="attributes+"]
+[,bash]
 ----
-include::get-started:attachment$three-brokers/docker-compose.yml[]
+docker exec -it redpanda-0 rpk topic describe logins \
+  -X user=Sasha \
+  -X pass=sashapassword \
+  -X sasl.mechanism=SCRAM-SHA-256 \
+  -X brokers=redpanda-0:9092
 ----
-====
-
-. Run the following in the directory where you saved the `docker-compose.yml` file:
 +
-```bash
-docker compose up -d
-```
+You are not authorized to view the topic.
 +
 [.no-copy]
 ----
-[+] Running 8/8
-⠿ Network redpanda_network                  Created  0.0s
-⠿ Volume "redpanda-quickstart_redpanda-0"   Created  0.0s
-⠿ Volume "redpanda-quickstart_redpanda-1"   Created  0.0s
-⠿ Volume "redpanda-quickstart_redpanda-2"   Created  0.0s
-⠿ Container redpanda-0                      Started  0.3s
-⠿ Container redpanda-console                Started  0.6s
-⠿ Container redpanda-1                      Started  0.7s
-⠿ Container redpanda-2                      Started  0.7s
+SUMMARY
+=======
+NAME        logins
+PARTITIONS  0
+ERROR       TOPIC_AUTHORIZATION_FAILED: Not authorized to access topics: [Topic authorization failed.]
+
+CONFIGS
+=======
+config response contained error: TOPIC_AUTHORIZATION_FAILED: Not authorized to access topics: [Topic authorization failed.]
 ----
 
---
-======
+. Access the `transactions` topic.
++
+[,bash]
+----
+docker exec -it redpanda-0 rpk topic describe transactions \
+  -X user=Sasha \
+  -X pass=sashapassword \
+  -X sasl.mechanism=SCRAM-SHA-256 \
+  -X brokers=redpanda-0:9092
+----
++
+You have access to this topic.
+
```

Comment:
I couldn't access. I'm in the 3 broker cluster in terminal, but in UI I'm in a 1 broker cluster: 
```
C:\Users\miche\docker-compose\three-brokers>docker exec -it redpanda-0 rpk topic describe transactions \
  -X user=Sasha \
  -X pass=sashapassword \
  -X sasl.mechanism=SCRAM-SHA-256 \
  -X brokers=redpanda-0:9092
unable to request topic metadata: broker closed the connection immediately after a request was issued, which happens when SASL is required but not provided: is SASL missing?

What's next:
    Try Docker Debug for seamless, persistent debugging tools in any container or image → docker debug redpanda-0
    Learn more at https://docs.docker.com/go/debug-cli/

C:\Users\miche\docker-compose\three-brokers>
```

Reply:
Let's have a call to run through the steps.